ui/src/lib.ts

```diff
@@ -12,33 +13,37 @@ const formatDate: Function = (date: Date): string => {
   ].join("-");
 };
 
-const formatDateMinusOneDay: Function = (date: Date): string => {
+const dateMinusOneDay: Function = (date: Date): Date => {
   // subtract a day from the given date
```

Comment:
I think we can remove this comment since the naming of the function and logic inside are straight forward to me. What do you think?

Reply:
Updated here 
8e55de2